cypress/support/commands.ts

```diff
@@ -530,6 +530,24 @@ export const createEndpoint = (
   return cy.request('POST', 'api/v2/notificationEndpoints', endpoint)
 }
 
+// helpers
+// Re-query elements that are found 'detached' from the DOM
+// https://github.com/cypress-io/cypress/issues/7306
+export const clickAttached = (subject?: JQuery<HTMLElement>) => {
```

Comment:
and some more magic. is there a forum where we can tell everyone "if xyz starts breaking, use clickAttached instead"?

Reply:
specifically, if xyz starts breaking _because_ its been detached from the DOM for sure.  I'm hosting a lunch and learn to disseminate all this info.  Also, this PR can act as an artifact on what changed and why.